### PR TITLE
4.7.0

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,31 @@
...
+### Changed
+- [[#1183](https://github.com/sonata-project/SonataUserBundle/pull/1183)]
+  SonataEasyExtendsBundle is now optional, using SonataDoctrineBundle is
+preferred ([@jordisala1991](https://github.com/jordisala1991))
+- [[#1196](https://github.com/sonata-project/SonataUserBundle/pull/1196)]
+  Changed API routing config definitions in order to use service ids instead of
+controller paths. ([@phansys](https://github.com/phansys))
+
+### Deprecated
+- [[#1183](https://github.com/sonata-project/SonataUserBundle/pull/1183)] Using
```
Comment (jordisala1991) [inline CHANGELOG.md:15]: This deprecation should be changed on the file too

Comment (greg0ire) [reply CHANGELOG.md:15]: Damn, I always forget it :sweat_smile:

Comment (greg0ire) [reply CHANGELOG.md:15]: Oh and there is the upgrade guide

**Comment by phansys** [comment]
Please, wait before merging this. I'll try to provide 2 PRs in the next hour which I'd like to be included in this release.

**Comment by phansys** [comment]
>:eyes:

I'd like to include these PRs if they could be merged before this release:
* #1200;
* #1201.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,31 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [4.7.0](https://github.com/sonata-project/SonataUserBundle/compare/4.6.0...4.7.0) - 2020-07-27
+### Changed
+- [[#1183](https://github.com/sonata-project/SonataUserBundle/pull/1183)]
+  SonataEasyExtendsBundle is now optional, using SonataDoctrineBundle is
+preferred ([@jordisala1991](https://github.com/jordisala1991))
+- [[#1196](https://github.com/sonata-project/SonataUserBundle/pull/1196)]
```
Comment (phansys) [inline CHANGELOG.md:10]: IMO, this one should be considered pedantic, since this should not be perceived by final users.

**Comment by jordisala1991** [comment]
Those 2 Pr are merged now, are we ready to release? @phansys

**Comment by phansys** [comment]
> Those 2 Pr are merged now, are we ready to release? @phansys

Yes! Thank you for waiting :+1:

File: CHANGELOG.md (hunk 2 above)
Comment (greg0ire) [reply CHANGELOG.md:10]: Then why did you tag it at minor :sweat_smile: ?

Comment (greg0ire) [reply CHANGELOG.md:10]: That being said I agree with you, let's drop it.

Comment (phansys) [reply CHANGELOG.md]: My bad :facepalm:

**Pull request merged**